### PR TITLE
Send receive JSON

### PR DESCRIPTION
**Pull request by tyler000000**

### CODE REVIEW

File: backend/.erlang.mk/deps.log [hunk 1]
```diff
@@ -1 +1,2 @@
 /Users/tylercalabrese/Dev/cs21-platformer/backend/deps/cowboy
+/Users/tylercalabrese/Dev/cs21-platformer/backend/deps/jsx
```
Comment (rebecca-millerr) [inline backend/.erlang.mk/deps.log:2]: Is this like something that we're all gonna need to have our own local version of the path to it when we're running or what?

File: backend/.erlang.mk/recursive-deps-list.log [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/Users/tylercalabrese/Dev/cs21-platformer/backend/deps/cowboy
+/Users/tylercalabrese/Dev/cs21-platformer/backend/deps/cowlib
+/Users/tylercalabrese/Dev/cs21-platformer/backend/deps/jiffy
+/Users/tylercalabrese/Dev/cs21-platformer/backend/deps/ranch
```
Comment (rebecca-millerr) [inline backend/.erlang.mk/recursive-deps-list.log:4]: Same question here

**Review by rebecca-millerr — APPROVED** [review]
I don't understand sockets but looks good I think lol

File: backend/.erlang.mk/deps.log (hunk 1 above)
Comment (tyler000000) [reply backend/.erlang.mk/deps.log:2]: oops i should add logfiles to the gitignore lol

Dependencies install on their own when you run `make` or `make run`

**Pull request merged**